asyncscope.md

```diff
@@ -823,64 +832,68 @@ One mental model for this is a semaphore. It tracks a count of lifetimes and fir
 
 Another mental model for this is block syntax. `{}` represents the root of a set of lifetimes of locals and temporaries and nested blocks.
 
-Another mental model for this is a container. This is the least accurate model. This container is a value that does not contain values. This container contains a set of lifetimes that have not completed (a lifetime is not a value).
+Another mental model for this is a container. This is the least accurate model. This container is a value that does not contain values. This container contains a set of active senders (an active sender is not a value, it is a state).
 
 alternatives: `sender_scope`, `dynamic_scope`, `dynamic_lifetime`, `scope`, `lifetime`
 
-## `nest`
+## `nest()`
 
-This provides a way to extend the lifetime to include a sender. This does not allocate state, call connect or start. This is the basic operation for `async_scope`. `spawn` and `spawn_future` use `nest` to extend the scope and then allocate, connect and start.
+This provides a way to build a sender that, when started, extends the lifetime of the `async_scope` to include the given sender. This does not allocate state, call connect or call start. This is the basis operation for `async_scope`. `spawn()` and `spawn_future()` use `nest()` to extend the scope and then they allocate, connect and start the returned @_nest-sender_@.
 
 It would be good for the name to indicate that it is a simple operation (insert, add, embed, extend might communicate allocation, which this does not do).
 
-alternatives: `add`, `extend`, `embed`, `include`
+If this becomes a basis operation for a new concept, it might be good to use a name that would also work for things like `async_mutex`. _See_ [Uses in other concurrent abstractions]
 
-## `spawn`
+alternatives: `add()`, `extend()`, `embed()`, `include()`, `constrain()`, `apply()`
 
-This provides a way to start a sender that produces `void` and extend the lifetime of the `async_scope` to exceed the lifetime of the operation. This allocates, connects and starts the sender.
+## `spawn()`
+
+This provides a way to start a sender that produces `void` and extend the lifetime of the `async_scope` to exceed the lifetime of the operation. This allocates, connects and starts the given sender.
 
 It would be good for the name to indicate that it is an expensive operation.
 
-alternatives: `start`, `submit`, `enqueue`, `do`, `run`
+alternatives: `start()`, `submit()`, `enqueue()`, `do()`, `run()`
```

Comment:
In general, I'm a tad concerned about the amount of various alternatives this paper lists. I tend to think that has a negative effect; the review groups are not good at choosing names, so the more firm decisions you can make about naming, the better, if you ask me.